recipes/aiomultiprocess/run_test.py

```diff
@@ -0,0 +1 @@
+from aiomultiprocess import *
```

Comment:
I think this would be better by instead adding a `test` section to `meta.yaml`:
```yaml
test:
  imports:
    - aiomultiprocess.core
    - aiomultiprocess.pool
    - aiomultiprocess.scheduler
    - aiomultiprocess.types
```

Reply:
yep you are perfectly right! Thanks for the comment!